superdesk/utils.py

```diff
@@ -154,3 +155,11 @@ def compare_preferences(original, updates):
     removed = original_keys - updates_keys
     modified = {o: (original[o], updates[o]) for o in intersect_keys if original[o] != updates[o]}
     return added, removed, modified
+
+
+def sha(text):
```

Comment:
I keept sha so that we can change it later to more secure version without affecting all the code..